scripts/manual_assign.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import re
+import requests
+import json
+
+
+def table_to_number(table):
+    letter = table[0].upper()
+    num = table[1:]
+    return (ord(letter) - 65) * 15 + int(num)
+
+
+# numeric value to table number: 210 -> N15, 1 -> A1
+def number_to_table(number):
+    letter = chr((int(number) - 1)//15 + 65)
+    num = int(number) - (int(number) - 1)//15 * 15
+    return str(letter) + str(num)
+
+
+def available_tables():
+    all_tables = list(map(lambda x: number_to_table(x), list(range(1, 391))))
+    r = requests.get("http://127.0.0.1:5000/api/projects")
+
+    taken = re.findall('\"table_number\": \"(\w\d+)\"', r.text)
+    taken.sort()
+    open_tables = list(set(all_tables) - set(taken))
+    open_tables.sort()
+
+    print("Available tables: ")
+    print(open_tables)
+
+
+def add_project(table_number, project_url, project_name, attempted_challenges):
```

Comment:
Instead of re-adding the project, we can make a call to the DB to update a current project with a table_number.

Example (referenced from [this Stackoverflow answer](https://stackoverflow.com/questions/13710770/how-to-update-values-using-pymongo)):
```
mongo.db.projects.update_one({
  '_id': #This should be the _id of the project you're currently assigning a table_number to
},{
  '$set': {
    'table_number': #This should be the table_number you want to assign this project to
  }
}, upsert=False)
```